### PR TITLE
feat(atenlib): aten_expand supports -1 dim

### PR DESCRIPTION
**Pull request by titaiwangms**

### CODE REVIEW

**Review by gramalingam — APPROVED** [review]

**Comment by titaiwangms** [comment]
> Are there tests we can enable for this?

Thinking about parametrizing fx-export into runtime test.

**Comment by justinchuby** [comment]
Was there an OpInfo covering this case?

File: onnxscript/function_libs/torch_aten/ops/core.py [hunk 1]
```diff
@@ -2039,6 +2039,8 @@ def aten_expand(self: TTensor, size: TInt) -> TTensor:
     """expand(Tensor(a) self, SymInt[] size, *, bool implicit=False) -> Tensor(a)"""
 
     size = op.Cast(size, to=INT64.dtype)
+    # To support -1 dim.
+    size = op.Abs(size)
```
Comment (justinchuby) [inline onnxscript/function_libs/torch_aten/ops/core.py:2043]: I understand -1 becomes 1. Was that intended or was I missing something?

Comment (titaiwangms) [reply onnxscript/function_libs/torch_aten/ops/core.py:2043]: Only -1 becomes 1. Do you think comment is redundant?

**Pull request merged**